postgres/rdspostgres/rdspostgres.go

```diff
@@ -71,20 +85,19 @@ func init() {
 
 // OpenPostgresURL opens a new RDS database connection wrapped with OpenCensus instrumentation.
 func (uo *URLOpener) OpenPostgresURL(ctx context.Context, u *url.URL) (*sql.DB, error) {
-	cf := new(rds.CertFetcher)
-	vals := u.Query()
-	u.RawQuery = vals.Encode()
-
+	if uo.CertSource == nil {
+		return nil, fmt.Errorf("rdspostgres: URLOpener CertSource is nil")
+	}
 	password, _ := u.User.Password()
-	params := Params{
+	params := &Params{
 		Endpoint:  u.Host,
 		User:      u.User.Username(),
 		Password:  password,
 		Database:  u.RawPath,
```

Comment:
This needs the same `strings.TrimPrefix(u.Path, "/")` logic as in the other places.